pkg/generate/app/cmd/newapp.go

```diff
@@ -883,6 +884,15 @@ func (c *AppConfig) Run() (*AppResult, error) {
 			}
 		}
 	}
+	if c.SourceSecret != "" {
+		for i, obj := range objects {
+			if _, ok := obj.(*buildapi.BuildConfig); ok {
```

Comment:
instead of re-doing the type assertion below, just use the value that would be assigned to `_`

Reply:
updated in next push